src/elements/common/annotator-context/types.ts

```diff
@@ -60,7 +75,8 @@ export interface Metadata {
 }
 
 export interface AnnotationActionEvent {
-    annotation?: object;
+    annotation?: Object;
```

Comment:
Do we need to type out these fields?

Reply:
It's an object containing annotation updates. It may contain any number of its properties. I didn't want to change the already used approach.